### PR TITLE
Ignore display style to work on Android

### LINKED ISSUE #1: Error while updating property `display` on Android
Seems like Android native `<HorizontalScrollView>` element doesn't like to receive `display: 'block'` styles, and therefore is throwing an error.
image

I'm not sure what to do to fix that, since on iOS seems to work just fine

### PR DESCRIPTION
**Pull request by stefanoeb**
Fixes #1

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ function generateNewStylesheet(stylesheet) {
       else if (key === 'background') {
         newStyle.backgroundColor = value;
       }
+      else if (key === 'display') {
+        return newStyle
```
Comment (conorhastings) [inline src/index.js]: can you add semicolon, i'll try to get an eslint or convert to prettier soon to avoid these annoyances

**Review by conorhastings — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (stefanoeb) [reply src/index.js]: yep, in a couple hours I'll send a fix

**Comment by conorhastings** [comment]
sweet thanks

**Comment by conorhastings** [comment]
published as `1.1.0`

**Pull request merged**